### PR TITLE
Support details

### PR DESCRIPTION
**Pull request by AsalaKM**
Technical support ticket details and allow the manager to update it

- [x] View the details of ticket.
- [x] Update ticket and add new status or action for it.

**Relates #12**

### CODE REVIEW

File: public/js/updateTicket.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    fetch(`/support/${ticketNo}`, {
+      method: 'PUT',
+      credentials: 'same-origin',
+      headers: { 'Content-Type': 'application/json; charset=utf-8' },
+      body: JSON.stringify(data),
+    })
+      .then(response => response.json())
+      .then((response) => {
+        window.location = '/support';
+        console.log(response.res);
```
Comment (FarahZaqout) [inline public/js/updateTicket.js]: Remove console.log

File: src/controllers/supportDetails.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+};
+
+exports.put = (req, res, next) => {
+  const data = req.body;
+  updateTicket(data)
+    .then((response) => {
+      res.send(data);
+    })
+    .catch((err) => {
+      console.log(err);
```
Comment (FarahZaqout) [inline src/controllers/supportDetails.js]: remove console.log

File: src/model/queries/ticket.js [hunk 3]
```diff
@@ -88,11 +88,16 @@ const addTicket = (newTicket) => {
...
+    statusType, itEmployeeName, description, ticketNo,
+  } = data;
+
   const sql = {
-    text: 'UPDATE ticket SET status_type = $1, it_employee = $2 WHERE ticket_no = $3;',
-    values: [itEmployee, status, ticketNo],
+    text: 'UPDATE ticket SET status_type = $1, it_employee = $2, technical_desc = $3 WHERE ticket_no = $4;',
+    values: [statusType, itEmployeeName, description, ticketNo],
   };
+  console.log(sql);
```
Comment (FarahZaqout) [inline src/model/queries/ticket.js]: remove console.log

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

**Review by FarahZaqout — CHANGES_REQUESTED** [review]
There's still a console.log

**Review by FarahZaqout — APPROVED** [review]

**Review by SalwaMugh — APPROVED** [review]

**Pull request merged**